main.go

```diff
@@ -218,13 +221,20 @@ func NewGame(g *Game) {
 
 	// Add spawnpoints to the game
 	for _, e := range entities.Entities {
-		if e.Identifier == "Zombie" {
+		if e.Identifier == "Zombie" || e.Identifier == "Zombie_sprinter" || e.Identifier == "Zombie_big" {
```

Comment:
Meh... I hope we don't add even more types to the map soon... this long chain of ORs could've been avoided if all spawn points were the same entity only with a different property, then only the inner if-s (a few lines down from here) would've been needed. Whatever, this'll do for now but if we add another type IMO we should change this.

Reply:
Agree, I dont really like the current structure.